common/ts/publish-task.ts

```diff
@@ -39,14 +37,15 @@ export default async function publishTask(endpointType: EndpointType) {
     if (e instanceof TimeOutReachedError) {
       tl.warning(
         `Task '${
-          task.analysisId
+          taskReport.ceTaskId
```

Comment:
We don't have the `task` if a timeout error is thrown

Reply:
Good catch!